cypress/integration/blocks-image.js

```diff
@@ -93,5 +91,23 @@ if (Cypress.env('API') !== 'guillotina') {
     //   cy.wait(5000);
     //   cy.url().should('eq', Cypress.config().baseUrl + '/my-page');
     // });
+    it('Add image via upload', () => {
```

Comment:
@iFlameing this is a test for an existing document in edit view. The problem I ran into only occurred when I created a new document and added an image. That's a different test setup.

Reply:
@iFlameing did you see my comment here? ^^^

Reply:
I saw it that's why I have written another test in edit mode